crates/fluvio-spu-schema/src/produce/request.rs

```diff
@@ -36,6 +37,10 @@ pub struct ProduceRequest<R> {
 
     /// Each topic to produce to.
     pub topics: Vec<TopicProduceData<R>>,
+
+    #[fluvio(min_version = 7)]
```

Comment:
I think that you have to update this to 8 version, and then  bump `DEFAULT_API_VERSION` and `MAX_API_VERSION` in lines 53-54 from this file

```suggestion
    #[fluvio(min_version = 8)]
```

Reply:
Maybe we can use a `const PRODUCER_TRANSFORMATION_API = 8` and use it here and in all the other places where that is used